### PR TITLE
Use mongoDB query instead of ES when no setting provided

### PR DESCRIPTION
**Pull request by quaxsze**
fix https://github.com/etalab/data.gouv.fr/issues/804

### CODE REVIEW

**Comment by quaxsze** [comment]
Looking for a possible other way of coding this.

**Comment by quaxsze** [comment]
First draft of refactoring. WDYT @abulte @maudetes ?

**Comment by quaxsze** [comment]
@maudetes

File: udata/search/result.py [hunk 1]
```diff
@@ -48,14 +48,14 @@ def get_ids(self):
             return []
 
     def get_objects(self):
-        if not self._objects:
+        if not self.mongo_objects:
```
Comment (maudetes) [inline udata/search/result.py:51]: Called when mongo_objects is None because no result has been found, resulting in `get_ids` getting called and crashing.

**Review by maudetes — COMMENTED** [review]
Seems like a good compromise, to have working udata search without our search service architecture.

You need to add a `mongo_search` class method in reuses/search.py as well :)

File: udata/settings.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class Defaults(object):
     MONGODB_CONNECT = False  # Lazy connexion for Fork-safe usage
 
     # Search service configuration
-    SEARCH_SERVICE_API_URL = 'http://127.0.0.1:5000/api/1/'
+    SEARCH_SERVICE_API_URL = ''
```
Comment (maudetes) [inline udata/settings.py:32]: Use `None` instead?

```suggestion
    SEARCH_SERVICE_API_URL = None
```

**Review by maudetes — APPROVED** [review]
:clap:

**Pull request merged**